python/gps/algorithm/policy/tf_policy.py

```diff
@@ -0,0 +1,87 @@
+import pickle
+
+import numpy as np
+import tensorflow as tf
+
+from gps.algorithm.policy.policy import Policy
+
+
+class TfPolicy(Policy):
+    """
+    A neural network policy implemented in tensor flow. The network output is
+    taken to be the mean, and Gaussian noise is added on top of it.
+    U = net.forward(obs) + noise, where noise ~ N(0, diag(var))
+    Args:
+        obs_tensor: tensor representing tf observation. Used in feed dict for forward pass.
+        act_op: tf op to execute the forward pass. Use sess.run on this op.
+        var: Du-dimensional noise variance vector.
+        sess: tf session.
+        device_string: tf device string for running on either gpu or cpu.
+    """
+    def __init__(self, dU, obs_tensor, act_op, var, sess, device_string):
+        Policy.__init__(self)
+        self.dU = dU
+        self.obs_tensor = obs_tensor
+        self.act_op = act_op
+        self.sess = sess
+        self.device_string = device_string  # is it even worth running this on the gpu? Won't comm time dominate?
```

Comment:
For convoluational network policies, even with a batch size of 1, we've found it worth it to use a GPU.
Remove comment?